docs/sphinx_doc/en/source/tutorial/208-distribute.md

```diff
@@ -79,6 +79,12 @@ server.launch()
 server.wait_until_terminate()
 ```
 
+> For similarity, you can run the following command in your terminal rather than the above code:
```

Comment:
The arguments in `agentscope.init` haven't been considered in this bash command.